wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/*
+Plugin Name:  AB Content Alterer
+Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
+Version:      0.7

```

Comment:
You may want to consider using semver guidelines for versioning your code - [http://semver.org/](http://semver.org/)